### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### In this project I:

- Used Gitflow as required by the project description.
- Setup the project with webpack.
- Created the HTML structure for the project.
- Used ES6 modules, with import and export.
- Fixed linter errors.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
+                            <h3>Add your score</h3>
+                        </div>
+                        <form>
+                            <input id="name" type="text" placeholder="Your name"/>
+                            <input id="score" type="text" placeholder="Your score"/>
+                        </form>
+                        <button id="submit-btn" type="submit">Submit</button>
+                </div>
+            </div>
+        </section>
```
Comment (zieeco) [inline dist/index.html]: **[REQUIRED CHANGES]**

- Please, kindly ensure your design matches the wireframe below: 

*wireframe*
matchhh

*your design*
match

File: dist/index.html [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
+                    <div class="inner-left-header">
+                        <h3>Recent scores</h3>
+                        <button type="button">Refresh</button>
+                    </div>
+                    <div class="scores">
+                        <p>Paul: <span>100</span></p>
+                        <p>Paul: <span>70</span></p>
+                        <p>Paul: <span>125</span></p>
+                        <p>Paul: <span>150</span></p>
+                        <p>Paul: <span>120</span></p>
```
Comment (zieeco) [inline dist/index.html]: **[REQUIRED CHANGES]**

- Please, kindly ensure the populated names and scores are populated dynamically using javascript.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Paul-js-hub,


Great job with the leaderboard setup 👏 
You've done well implementing it, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Paul-js-hub 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedAgreeGIF


**_Great work implementing building the setup for this project 👏_** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**